src/main/java/mekanism/common/capabilities/energy/MinerEnergyContainer.java

```diff
@@ -45,7 +45,7 @@ public void updateEnergyPerTick() {
     public void updateMinerEnergyPerTick() {
         minerEnergyPerTick = super.getEnergyPerTick();
         if (tile.getSilkTouch()) {
-            minerEnergyPerTick = minerEnergyPerTick.multiply(MekanismConfig.general.minerSilkMultiplier.get());
+            minerEnergyPerTick = minerEnergyPerTick * (MekanismConfig.general.minerSilkMultiplier.get());
```

Comment:
You can make this just be `*=` and also remove the parenthesis around the silk multiplier